db/migrate/20200616170538_create_offers.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateOffers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :offers do |t|
+      t.string :message
+      t.belongs_to :helper, foreign_key: { to_table: :users }
```

Comment:
shouldn't we call it user and refer to the id in the user  table? since the helper is a user....

Reply:
By doing this it refers to the user table, but the key on the offer is helper. Just to avoid using too general terms and cause confusion if we end up having the same model doing two different things.

Reply:
ok then we go with that, we just continue working with helper!